.buildkite/release-pipelines/start-code-freeze.yml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+# Variables used in this pipeline are defined in `shared-pipeline-vars`, which is `source`'d before calling `buildkite-agent pipeline upload`
+
+steps:
+  - label: Start Code Freeze
+    plugins:
+      - $CI_TOOLKIT_PLUGIN
+    # The first client to implement releases in CI was Android so the automation works in that queue.
+    # We might want to move it to a leaner one in the future.
+    agents:
+        queue: android
```

Comment:
After looking at @iangmaia's recent work adding a release-publishing pipeline to one of our apps that is not yet open source, it occurred to me that starting the code freeze could run on the `tumblr-metal` agent to leverage the more secure read vs read-write agent, since clearly it does not need macOS

However, I'm not 100% sure of the what's exactly required and want to get testing the code freeze in CI. I'll leave that as something to investigate next.

Reply:
🤔 I might have to switch sooner 

![image](https://github.com/user-attachments/assets/17a0e43b-2d35-4328-989c-b618ab09cf06)

https://buildkite.com/automattic/simplenote-ios/builds/1087#0191bb69-078a-43fe-9053-0291750baf22